Test failure is expected because this PR is changing an EOL distro

```
Build cache for "noetic"
- trying to use local cache
- trying to fetch cache
- update cache
- removing invalid release package cache entries.
- fetch missing release manifests
......ERROR: In file '''/home/runner/work/rosdistro/rosdistro/indigo/distribution.yaml''': There is a change to a doc section of an EOLed distribution. Lines: [13671]
F
....................................................................................................................................................................................................................................................................................................................................................
======================================================================
FAIL: test_function (test.test_url_validity.TestUrlValidity)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/rosdistro/rosdistro/test/test_url_validity.py", line 353, in test_function
    self.assertFalse(detected_errors)
AssertionError: ["In file '''/home/runner/work/rosdistro/rosdistro/indigo/distribution.yaml''': There is a change to a doc section of an EOLed distribution. Lines: [13671]"] is not false
```

--- a/indigo/distribution.yaml
+++ b/indigo/distribution.yaml
@@ -13668,7 +13668,7 @@ repositories:
     doc:
       type: git
       url: https://github.com/ros-visualization/rqt_common_plugins.git
-      version: indigo-EOL
+      version: indigo-devel
     release:
       tags:
         release: release/indigo/{package}/{version}
@@ -13677,7 +13677,7 @@ repositories:
     source:
       type: git
       url: https://github.com/ros-visualization/rqt_common_plugins.git
-      version: indigo-EOL
+      version: indigo-devel
     status: developed
   rqt_console:
     doc:

